layers/+core/behavior/config.vim

```diff
@@ -0,0 +1,70 @@
+" call SpaceNeovimLoadFunc(expand('<sfile>:p'), 'func.vim')
+
+" Use system clipboard
+set clipboard=unnamed
+set noswapfile                      " Disable swap
+
+" Shorten the time before the vim-leader-guide buffer appears
+set timeoutlen=500
+set ttimeoutlen=0
+
+" UI {{{
+  set modeline                      " Read modelines from files
+  set laststatus=2
+  set list                          " Show invisible characters
+  set scrolloff=7                   " Set 7 lines to the cursor - when moving vertically using j/k
+  set splitright                    " Split on the right side
+  set splitbelow                    " Split below
+  set number                        " Show current line number
+  set relativenumber                " Enable relative line numbers
```

Comment:
Yeah that actually sounds good, to split up core behaviour that's needed and then some sensible opinionated defaults that we feel are good to have as a starting point :) Like the idea very much!